core/src/main/java/org/infinispan/remoting/transport/impl/MultiTargetRequest.java

```diff
@@ -167,6 +167,6 @@ protected void onTimeout() {
                                              .filter(Objects::nonNull)
                                              .map(Object::toString)
                                              .collect(Collectors.joining(","));
-      completeExceptionally(CLUSTER.requestTimedOut(requestId, targetsWithoutResponses));
+      completeExceptionally(CLUSTER.requestTimedOut(requestId, timeoutMessagePosfix(targetsWithoutResponses)));
```

Comment:
I would change the `requestTimedOut` to include the time (as string) and use `Util.prettyPrintTime()` to get a "nice" string